src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --hover-color: #00acdc;
+}
+
+.container {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  align-content: space-around;
   margin: 0;
 }
+
+.logo.png {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  width: 1200px;
+  justify-content: flex-end;
+  list-style: none;
+  padding: 0;
+  margin-left: 50px;
+}
+
+.nav_item {
```

Comment:
It seems there is a typo in the class name `.nav_item`. According to the naming convention used in the rest of the file, it should be `.nav__item`.

Reply:
Corrected the typo in the class .nav__item